### ISSUE TITLE
Error in Game Control Panel?

### ISSUE THREAD

**Issue by sage944** [vasl-developers/vasl#1448]
There seems to be an issue with the game control panel.  There appears to be two panels, one (behind) has the dice buttons.  This cannot be "switched to".  I use another method for dice but this should be brought to someone's attention.  Am I doing something wrong?  I have attached a JPEG screen shot.  You can see the issue on the right-hand  side of the left image.
![VASL Error](https://github.com/vasl-developers/vasl/assets/93237708/ed557526-af48-41d8-a23f-a6b651bdb395)

**Comment by sage944** [comment]
Good Morning!

Yes, removing the MDX extensions fixed it.  These are older anyway so maybe not needed?  The interface seems to operate without them.  I have moved them so they will not be read.

Best,
Jeffrey D. Fisher
Omaha, NE USA


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by derimmer** [comment]
There may well be some old .mdx extensions that have continuing value. 

Personally, I would load them on an "as and when needed" basis rather than having a whole bunch loaded that you don't need.

**Comment by sage944** [comment]
Yeah, good point.  That is why I moved them instead of deleting them.  I intended to isolate the cause and then delete it/them.

Best,
Jeffrey D. Fisher
Omaha, NE USA


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
